packages/react-heartwood-components/src/components/BigCalendar/BigCalendar.js

```diff
@@ -382,6 +389,8 @@ class BigCalendar extends Component<Props, State> {
 					totalHorizontalPages={totalHorizontalPages}
 					onHorizontalPageNext={this.handleHorizontalPageNext}
 					onHorizontalPageBack={this.handleHorizontalPageBack}
+					onSelectDate={this._setStartDate}
+					onDateToToday={this.handleDateToToday}
```

Comment:
Hmm, nit: `onDateToToday` -> `onSetDateToToday` to preserve the action verb in `on`-prefixed props.

Don't feel crazy strongly about this but I found `onDateToToday` somewhat confusing on first read

Reply:
👍makes sense!